content/v2.0/write-data/sample-data/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Sample data
```

Comment:
Can we change this and all subsequent references to Demo Data please?

Reply:
Good suggestion, but there was a few reasons for doing this:

1. Findability - people search for "sample data" more than "demo data."
2. Differentiation – This PR includes pages for both "Sample data" and "Demo data":
    - **Sample data:** Semi-static data sets we provide in the documentation to highlight/walkthrough specific features.
    - **Demo data:** Buckets provided in InfluxDB for introducing and demo-ing the product.

Reply:
ok! sorry I thought this was documentation of demodata feature

Reply:
No worries. This PR both consolidates the existing sample data sets into a single doc and introduces a new demo data doc.